src/static/stilyagi/assets/styles/pages/home.css

```
@@ -484,6 +490,78 @@ top: 145px;
+  }
+
+  .poster-tiles .poster-tile:nth-child(3) {
+    animation-delay: 400ms;
+  }
+
+  .poster-tiles .poster-tile:nth-child(4) {
+    animation-delay: 600ms;
+  }
+
+  @keyframes tile-chase {
+    from {
```

Stylelint (17.14.0)

[error] Expected "from" to be "0%" (keyframe-selector-notation)

(keyframe-selector-notation)